### ISSUE TITLE
[A11Y]

### ISSUE THREAD

**Issue by erikkroes** [Tiqr/eduid-app-ios#254]
## Finding and location

The info fields like those on the “your personal info”-page are not recognized as being interactive. The same goes for the expendables on the activity-page.

![](https://github.com/user-attachments/assets/7a0c2d9b-8992-41d2-bcff-69d17bafedd4)

## Problem

Users, for example those with screen readers, that do not know items are interactive are unable to use them.

## Solution

Make use `accessibilityTraits` in iOS to identify interactive items.

## Resources
- https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html
- https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html
- https://developer.apple.com/documentation/objectivec/nsobject-swift.class/accessibilitytraits
